shell/platform/fuchsia/BUILD.gn

```diff
@@ -81,6 +81,7 @@ if (using_fuchsia_sdk) {
       "dart_runner:dart_jit_${product_suffix}runner",
       "flutter:flutter_aot_${product_suffix}runner",
       "flutter:flutter_jit_${product_suffix}runner",
+      "flutter:flutter_runner_tests",
```

Comment:
Did you mean to have this part of this PR?

Reply:
I'm not sure -- I added this to enable building the tests by default, since they are mostly working now... but if you think it makes more sense in a different PR I can move it

Reply:
We explicitly build the flutter_runner_tests target as part of the LUCI recipe. With LTO on this additional target can add a significant amount of time to the build, so let's avoid adding it for now.

Reply:
(I will likely be switching the fml_tests target to be explicit in the same way)